docs/rfc/rfc-1.adoc

```diff
@@ -459,11 +464,12 @@ not disincentivize it for waiting for enough redemption requests to accumulate,
 the redemption timeout should be long enough, for example, 72 hours and ideally
 even longer in the early days of the system.
 
-A timed-out redemption request is considered a <<heartbeat,heartbeat>> failure
-and the wallet is ordered to move the BTC to another random wallet. The wallet
-that timed out on processing redemption request can not be requested for another
-redemption. Operators are slashed for a timeout but they continue to earn
-rewards.
+A timed-out redemption request is considered a number consecutive
+<<heartbeat,heartbeat>> failures equal to `missed_redemption_heartbeats` and
+depending on `consecutive_failed_heartbeats` the wallet is ordered to move the
+BTC to another random wallet. The wallet that timed out on processing
+redemption request can not be requested for another redemption. Operators are
+slashed for a timeout but they continue to earn rewards.
```

Comment:
Final conclusion is that we'll be always triggering wallet closure and moving ETH funds in case of redemption request timeout. I'm marking this conversation as resolved.